### PR TITLE
CUSTOM_MENU_COORDINATES to SET_GAMEPLAY_CAM_MOTION_BLUR_SCALING_THIS_…

### PR DESCRIPTION
**Pull request by rainmad0**
…UPDATE

Fixed wrong native name. 
Added native name.

### CODE REVIEW

**Review by PsychoShock — CHANGES_REQUESTED** [review]
Don't forget to add alias for the old names

File: CAM/SetGameplayCamMotionBlurScalingThisUpdate.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 ns: CAM
 aliases: ["0x487A82C650EB7799"]
 ---
-## CUSTOM_MENU_COORDINATES
+## SET_GAMEPLAY_CAM_MOTION_BLUR_SCALING_THIS_UPDATE
 
 ```c
 // 0x487A82C650EB7799 0xA6E73135
-void CUSTOM_MENU_COORDINATES(float p0);
+void SET_GAMEPLAY_CAM_MOTION_BLUR_SCALING_THIS_UPDATE(float p0);
```
Comment (PsychoShock) [inline CAM/SetGameplayCamMotionBlurScalingThisUpdate.md:9]: `p0` is the scaling.

File: CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 ns: CAM
 ---
-## _0x0225778816FDC28C
+## SET_GAMEPLAY_CAM_MAX_MOTION_BLUR_STRENGTH_THIS_UPDATE
 
 ```c
 // 0x0225778816FDC28C 0x1126E37C
-void _0x0225778816FDC28C(float p0);
+void SET_GAMEPLAY_CAM_MAX_MOTION_BLUR_STRENGTH_THIS_UPDATE(float p0);
```
Comment (PsychoShock) [inline CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md:8]: `p0` is the max strenght

**Comment by AvarianKnight** [comment]
You can't add aliases there would be collisions.

File: CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md (hunk 2 above)
Comment (PsychoShock) [reply CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md:8]: Please re-name the `p0` for `maxStrenght`

File: CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 ```
 
 ```
 some camera effect that is (also) used in the drunk-cheat, and turned off (by setting it to 0.0) along with the shaking effects once the drunk cheat is disabled. Possibly a cinematic or script-cam version of _0x487A82C650EB7799  
 ```
 
 ## Parameters
-* **p0**: 
+* **p0**: the max strenght
```
Comment (PsychoShock) [inline CAM/SetGameplayCamMaxMotionBlurStrenghtThisUpdate.md:16]: Same for here.

File: CAM/SetGameplayCamMotionBlurScalingThisUpdate.md (hunk 1 above)
Comment (PsychoShock) [reply CAM/SetGameplayCamMotionBlurScalingThisUpdate.md:9]: Please re-name the `p0` for `scaling`

File: CAM/SetGameplayCamMotionBlurScalingThisUpdate.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
 ```
 
 ```
 some camera effect that is used in the drunk-cheat, and turned off (by setting it to 0.0) along with the shaking effects once the drunk cheat is disabled.  
 ```
 
 ## Parameters
-* **p0**: 
+* **p0**: the scaling.
```
Comment (PsychoShock) [inline CAM/SetGameplayCamMotionBlurScalingThisUpdate.md:17]: Same for here.

**Review by PsychoShock — CHANGES_REQUESTED** [review]

**Pull request closed without merging**